### ISSUE TITLE
Can't install the module 

### ISSUE THREAD

**Issue by Superwash** [Z3nner/lancer-weapon-fx#9]
Hey since I updated to Foundry v10, it wont let me even download the module. Here is a screen shot of the error log.
![lancer error screenshot](https://user-images.githubusercontent.com/107323079/189601220-8ec41a95-5983-4e43-9cb4-17526fbd97fc.png)

**Comment by dodgepong** [comment]
The module has not been updated for Foundry v10 yet. Fortunately those errors are simply deprecation warnings and I can't think of anything off the top of my head in the module that wouldn't just be a deprecation issue.

It's worth noting that the Lancer Foundry system doesn't even have a v10-compatible release yet, so you're really better off sticking to v9 until more modules have been updated.

**Comment by Nachtrose** [comment]
Not sure, if this is V10 specific, but trying to install it i get this error:

![image](https://user-images.githubusercontent.com/81585420/192112451-461baf61-83f9-4732-b059-457642d804d5.png)

**Comment by Z3nner** [comment]
Please see dogepongs prior response. The system this module is intended for (Lancer) is not V10 compatible. This module will be updated some time after its parent system is updated to support V10.

**Comment by Nachtrose** [comment]
> Please see dodgepongs prior response. The system this module is intended for (Lancer) is not V10 compatible. This module will be updated some time after its parent system is updated to support V10.

Yeah, i saw it. But he was reacting to a post just with a lot of warnings.
The error i posted mentions `lang/en.json` missing. Which looking through the code is really missing. So i was making sure if it's more or less the same issue.

**Comment by Z3nner** [comment]
The module is compatible with V10 now. Are you still having any issues?
